### ISSUE TITLE
Pivot Table doesn't consider metric currency for column width

### ISSUE THREAD

**Issue by Jockxtar** [apache/superset#31377]
## Screenshot
![imagen](https://github.com/user-attachments/assets/8aa48e5f-a9e2-447a-95e9-81fa7f5ea7db)
![imagen](https://github.com/user-attachments/assets/7d0d0547-06ec-4280-a8f7-5e38b7b93508)

## Description
When adding metrics with currency to a pivot table chart, the column width is calculated based on the size of the number, but it doesn't account for the additional character of the currency symbol, forcing the symbol to be truncated into an additional line, which increases the rows' height.

## Design input
I suggest to increase columns' width if the metric that it represents has any currency at all.

**Comment by rusackas** [comment]
You might be able to fix this with a CSS one-liner of `white-space: nowrap;`

**Comment by rusackas** [comment]
@Vitor-Avila @sadpandajoe think any of us should open a short PR here? Not sure if anyone's been too concerned about this.

**Comment by sadpandajoe** [comment]
I don't think this is that high priority but @Vitor-Avila may have a different opinion about it.

**Comment by Vitor-Avila** [comment]
@rusackas there you go: https://github.com/apache/superset/pull/31414

As you know I'm not too familiar with frontend, so while I tested locally, it would be good to double check if that's the right place to add it.
